hoodie-client/src/test/java/com/uber/hoodie/io/strategy/TestHoodieCompactionStrategy.java

```diff
@@ -110,8 +110,8 @@ public void testPartitionAwareCompactionSimple() {
     sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
     DayBasedCompactionStrategy strategy = new DayBasedCompactionStrategy();
     HoodieWriteConfig writeConfig = HoodieWriteConfig.newBuilder().withPath("/tmp").withCompactionConfig(
-        HoodieCompactionConfig.newBuilder().withCompactionStrategy(strategy).withTargetIOPerCompactionInMB(400).build())
-        .build();
+        HoodieCompactionConfig.newBuilder().withCompactionStrategy(strategy).withTargetPartitionsPerCompaction(2)
+            .build()).build();
```

Comment:
Can you add a deterministic test-case where there are many operations per partitions and the day based compaction logic does the correct thing.